examples/lifecycleDemo.html

```diff
@@ -183,15 +271,27 @@ <h1>Lifecycle Events Log:</h1>
                     // Simulating delay of 500ms
                     window.setTimeout(() => {
                         myClientApp.lifecycle.stopped();
-
-                        myClientApp.alerting.showToastPopup(
-                            lifecycleStatusMessageTitle,
-                            'App Stopped (500ms delay)', {
-                                id: lifecycleStatusMessageId,
-                                type: 'error',
-                                showCloseButton: true
-                            }
-                        );
+                        if (requireManualToastDismissal) {
+                            myClientApp.alerting.showToastPopup(
+                                lifecycleStatusMessageTitle,
+                                `App Stopped (500ms delay):lifecycleStatusMessageId-stop-${count}`, {
```

Comment:
This is a different format than the rest of the app messages.  Still contains the `(500ms delay)` string.

Reply:
whoops.  nevermind.  Was comparing with focus/blur.  Disregard this comment.